package.json

```diff
@@ -84,31 +28,19 @@
       "git add"
     ]
   },
-  "main": "lib/agent.js",
   "repository": {
     "type": "git",
     "url": "git+ssh://git@github.com/userdive/agent.js.git"
   },
   "scripts": {
-    "build": "run-p build:*",
-    "build:bundle": "webpack --env=production && size-limit",
-    "build:doc": "typedoc --out docs/ src/ && touch docs/.nojekyll",
-    "build:ts": "tsc",
+    "build": "lerna run build",
     "ci:sauce": "cross-env CI_MODE=sauce npm run test",
-    "clean": "rimraf coverage lib docs",
-    "coverage": "codecov -f ./coverage/lcov.info",
-    "lint": "run-p lint:*",
-    "lint:ts": "tslint {src,test}/**/*.ts",
-    "postbuild": "cp robots.txt lib/build/.",
-    "prebuild": "npm run clean",
-    "precommit": "lint-staged",
-    "prepare": "npm run build",
-    "test": "karma start --single-run"
-  },
-  "size-limit": [
-    {
-      "path": "lib/build/agent.js",
-      "limit": "8 KB"
-    }
-  ]
+    "clean": "lerna clean --yes && lerna run clean",
+    "coverage": "lerna run coverage",
```

Comment:
`run <script> [args..]    Run an npm script in each package that contains that script.`